areas.yaml

```diff
@@ -257,3 +257,13 @@ areas:
     status:
       - accepting_contributions
       - active
+
+  - name: "Semantic Conventions: Container"
+    project: "N/A"
+    board: "N/A"
+    labels:
+      - area:container
+      - area:oci
+    status:
+      - accepting_contributions
```

Comment:
@open-telemetry/semconv-container-approvers is the group still active and interested in contributions? Please let me know and I'll update the PR to reflect actual status. 

Also if you a project board or any info about the project, could you please share the link?